src/main/java/hng_java_boilerplate/user/serviceImpl/UserServiceImpl.java

```diff
@@ -158,6 +172,40 @@ private void createPasswordResetTokenForUser(User user, String token) {
         passwordResetTokenRepository.save(newlyCreatedPasswordResetToken);
     }
 
+    @Override
+    public ResponseEntity<String> resetPassword(String token, ResetPasswordDto passwordDto) {
```

Comment:
This won't return a JSON back, it'll return a string

Reply:
I don't think mail sending endpoint return JSON, it just returns a string successful mail sent